### ISSUE TITLE
The term "visit" may be confusing when there are multiple images at the same time point

### ISSUE THREAD

**Issue by BryanGuillaume** [NISOx-BDI/SwE-toolbox#127]
The term `visit` in the toolbox may be confusing for repeated measures analysis when there are several contrast images from the same time point. Some users may want to use the same visit category for two contrast images that are from the same time points.

Therefore, we could rename this term. However, it does not seem easy to rename it in SwE.mat due to retro-compatibility. Thus, I would simply rename this in the model specification module of the toolbox and other places in the UI where the term visit appears. For example, we could use instead `Within-subject image labels`,  `Within-subject image categories` or else. Any other solutions to this issue or renaming suggestions are very welcome.

**Comment by nicholst** [comment]
Agreed this is confusing.  Best I can think of is "Visit or within-subject
condition" but then we have to be explicit that conditions are unique, i.e.
a subject can never have more than one scan per condition.
